src/lib.rs

```diff
@@ -174,6 +175,7 @@ const CERTIFICATE_FILE_NAMES: &[&str] = &[
     "/etc/pki/tls/cacert.pem",                           // OpenELEC (a media center Linux distro)
     "/etc/ssl/cert.pem",                                 // Alpine Linux
     "/opt/etc/ssl/certs/ca-certificates.crt", // Entware, https://github.com/rustls/openssl-probe/pull/21
+    "/etc/ssl/certs/cacert.pem", // OpenHarmony, https://developer.huawei.com/consumer/cn/doc/harmonyos-faqs/faqs-network-41
```

Comment:
nit: can we use the `en` translation of this page:

```suggestion
    "/etc/ssl/certs/cacert.pem", // OpenHarmony, https://developer.huawei.com/consumer/en/doc/harmonyos-faqs/faqs-network-41
```